src/game.cpp

```diff
@@ -2655,6 +2655,10 @@ void Game::openInventory()
 	InventoryLocation inventoryloc;
 	inventoryloc.setCurrentPlayer();
 	current_formspec->setFormSpec(fs_src->getForm(), inventoryloc);
+
+	if (client->moddingEnabled()) {
+		client->getScript()->on_open_inventory();
```

Comment:
better usage, move this before  `current_formspec->setFormSpec` and return a bool, if false, open the standard formspec else don't open